Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -419,35 +419,30 @@ private void AddFile(string fileName, string packageDirPath, string suffix)
             _logger.LogDebug("Add file entry processed.");
         }
 
-        private void ProcessModifiedFiles(string packageDirPath, string suffix,
+        private void ProcessModifiedFiles(string packageDirPath, string suffix, TemporaryDirectory tempDiffDir,
             CancellationToken cancellationToken)
         {
             _logger.LogDebug("Processing diff modified files...");
 
-            using (var tempDiffDir = new TemporaryDirectory(_temporaryData.GetUniquePath()))
+            _modifiedFilesStatusReporter.OnProgressChanged(0.0, "Installing package...");
+
+            for (int i = 0; i < _diffSummary.ModifiedFiles.Length; i++)
             {
-                _logger.LogTrace("tempDiffDir = " + tempDiffDir.Path);
+                cancellationToken.ThrowIfCancellationRequested();
 
-                _modifiedFilesStatusReporter.OnProgressChanged(0.0, "Installing package...");
+                var entryName = _diffSummary.ModifiedFiles[i];
 
-                for (int i = 0; i < _diffSummary.ModifiedFiles.Length; i++)
+                if (!entryName.EndsWith("/"))
                 {
-                    cancellationToken.ThrowIfCancellationRequested();
-
-                    var entryName = _diffSummary.ModifiedFiles[i];
-
-                    if (!entryName.EndsWith("/"))
-                    {
-                        PatchFile(entryName, packageDirPath, suffix, tempDiffDir);
-                    }
-
-                    _modifiedFilesStatusReporter.OnProgressChanged(
-                        (i + 1) / (double) _diffSummary.ModifiedFiles.Length, "Installing package...");
+                    PatchFile(entryName, packageDirPath, suffix, tempDiffDir);
                 }
 
-                _modifiedFilesStatusReporter.OnProgressChanged(1.0, "Installing package...");
+                _modifiedFilesStatusReporter.OnProgressChanged(
+                    (i + 1) / (double) _diffSummary.ModifiedFiles.Length, "Installing package...");
```

Comment:
If there won't be any modified files, you will get a division by 0.

Reply:
If there are no modified files then this code is never reached.